style.css

```diff
@@ -389,3 +393,187 @@ button:disabled {
   border: 1px solid #c1c7d0;
   color: #5e6c84;
 }
+
+@media only screen and (min-width: 768px) {
+  .humbarger {
+    display: none;
+  }
+
+  #navbar-menue {
+    display: inline-block;
+  }
+
+  #navbar-menue ul {
+    list-style-type: none;
+    display: flex;
+    gap: 32px;
+  }
+
+  #navbar-menue ul li a {
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  #phone {
+    height: 72px;
+    padding-left: 155px;
+    padding-right: 154px;
+  }
+
+  #bg {
+    height: 846px;
+    padding-left: 34.6px;
+    padding-right: 8px;
+    padding-top: 49px;
+  }
+
+  #bio {
+    height: 721px;
+    padding-left: 225px;
+    padding-right: 252px;
+    padding-top: 241px;
+    background-image: url('/images/Header-bg-desktop.png');
+  }
+
+  /* works */
+  #works {
+    padding-top: 142px;
+    padding-left: 142px;
+    padding-right: 142px;
+  }
```

Comment:
- Hey, good job with the styles, however as in terms of using your selectors in applying your styles, it is best to use classes to add styling as IDs are meant to be used ideally in Javascript as these are unique selectors so best used in JS instead of using them to apply styles as in the [HTML & CSS Best Practises](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). Kindly use classes to add your styling. 

:point_right: *this change is applicable to every part of your CSS where you've used the id selector to add styles*